src/Stratis.FederatedPeg.Features.FederationGateway/Wallet/FederationWalletManager.cs

```diff
@@ -132,7 +139,9 @@ public class FederationWalletManager : IFederationWalletManager
             IDateTimeProvider dateTimeProvider,
             IFederationGatewaySettings federationGatewaySettings,
             IWithdrawalExtractor withdrawalExtractor,
-            IBroadcasterManager broadcasterManager = null) // no need to know about transactions the node broadcasted
+            ICrossChainTransferStore crossChainTransferStore = null,    // Not required for store tests.
```

Comment:
If the node has been initialized properly, this should never be null.
Can you please remove this `= null`?

Reply:
This will add code to the tests but sure.